cmd/frontend/internal/search/httpapi/export.go

```diff
@@ -12,6 +13,11 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
+// search-jobs_<job-id>_2020-07-01_150405
+func filename(jobID int) string {
+	return fmt.Sprintf("search-jobs_%d_%s.csv", jobID, time.Now().Format("2006-01-02_150405"))
```

Comment:
call sites already append an extension. Maybe rename function to indicate it's filename prefix?
```suggestion
	return fmt.Sprintf("search-jobs_%d_%s", jobID, time.Now().Format("2006-01-02_150405"))
```